content/20.pharmaceuticals.md

```diff
@@ -769,7 +772,7 @@ As more evidence becomes available, the potential for existing and novel therapi
 **Mechanism of Action for Potential Therapeutics**
 Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
 Here, the relationship between the virus and several therapeutics described above are visualized.
-](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
+](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1} <!--Step #5 is not labeled-->
```

Comment:
@nilswellhausen would you be able to add a 5 into this? Sorry I missed it at review!

Reply:
![COVID life cycle and drugs (1)](https://user-images.githubusercontent.com/62564517/108059071-f665d580-7022-11eb-95d9-29f2c62a24c0.png)

Reply:
Great! It seems like step "9" should be "8" though. Also, the circles around 5-7 are much bigger than the others (that's not a big deal, so we can probably fix it in the revision.)

Reply:
oh wow I just saw that the circles just got all messed up. Let me fix that really quick. Not sure what happened there as it looked normal on my end.

Reply:
![COVID life cycle and drugs (2)](https://user-images.githubusercontent.com/62564517/108085902-d0046200-7043-11eb-9096-26d1b1988daa.png)

Reply:
Thank you! Looks awesome!